specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/computeRPCommon.json

```diff
@@ -1009,6 +1011,10 @@
           "description": "The managed disk parameters.",
           "$ref": "#/definitions/ManagedDiskParameters"
         },
+        "sourceResource": {
```

Comment:
you should add or edit an example with this property if applicable. https://github.com/Azure/azure-rest-api-specs/tree/compute-2024-03-01/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/examples